script/tool/lib/src/podspec_check_command.dart

```diff
@@ -171,7 +172,8 @@ class PodspecCheckCommand extends PackageLoopingCommand {
         return false;
       }
       final String filePath = entity.path;
-      return path.extension(filePath) == '.swift';
+      return entity.basename != 'Package.swift' &&
```

Comment:
Shouldn't we check the path to make sure it's at the root of the module? In theory someone could have a `Package.swift` as an actual source file within the plugin (although I guess it's very unlikely that even if they did it would be the *only* Swift file).

Reply:
Sure, can do